### DISCUSSION TITLE
recommended pattern for loading indicator

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#5873] [Q&A]
hi, i'm curious what patterns you would recommend for a loading indicator/spinner?

i had a look at the react spectrum ["pending" button variant](https://react-spectrum.adobe.com/react-spectrum/Button.html#pending), and from what i understand this combines both an indeterminate progressbar with an aria live region (?). would appreciate any guidance, thanks!

**Comment by snowystinger** [comment]
Thanks for the discussion. Can you say a little more about your use case? I am assuming that you aren't trying to do a button?
If you are, then I think that's a good example to follow.
